pktables.py

```diff
@@ -9,7 +9,7 @@
 
 
 PACKETKEY = os.environ.get("PACKET_API_AUTH_TOKEN", os.environ.get("PACKETKEY"))
-PROJECTID = os.environ.get("PACKET_PROJECT", os.environ.get("PROJECTID"))
+PROJECTID = os.environ.get("PACKET_PROJECT_ID", os.environ.get("PROJECTID"))
```

Comment:
please throw it inside another layer of `os.environ.get` instead of renaming. IDK if anyone ever used the `PACKET_PROJECT` version but might as well avoid pain if possible.